backend/lib/routes/index.js

```diff
@@ -0,0 +1,53 @@
+const { Api } = require('../controllers');
+const userRoute = require('./users');
+const tagsRoute = require('./tags');
+const deepmerge = require('../utils/deepmerge');
+
+const routes = [
+    {
+        method: 'get',
+        path: '/',
+        controller: Api,
+        args: { public: true },
+    },
+    userRoute,
+    tagsRoute,
+];
+
+/**
+ * Makes the child object inherit the args of the parent object, but keeps the overwrites
+ * @param {Object} parent parent who's args will be inherited
+ * @param {Object} child child who will inherit the args
+ * @returns {undefined}
+ */
+const inheritArgs = (parent, child) => {
+    child.args = deepmerge(parent.args||{}, child.args||{});
+};
+
+/**
+ * Route binder, used to bind routes to the httpServer
+ * @param {Object} http httpServer that the routes should be bound on
+ * @param {Object} route the route that should be bound
+ * @param {String} [parentPath=''] the base path of the route
+ * @returns {undefined}
+ */
+const bindRoute = (http, route, parentPath = '') => {
+    const localPath = parentPath.concat(route.path);
```

Comment:
Shouldn't we check if the path starts with or without a `/`, for example: `/api` + `missing-dash`, this would result in `/api-missing-dash`, is this expected or not? Perhaps test worthy situation 😉.